mm.c

```diff
@@ -44,11 +44,12 @@ team_t team = {
 #define CHUNK (1<<12) /* extend heap by this amount (bytes) */
 #define STATUS_BIT_SIZE 3 // bits
 #define HDR_FTR_SIZE 2 // in words
-#define HDR_SIZE 1 // in words 
-#define FTR_SIZE 1 // in words 
+#define HDR_SIZE 1 // in words
+#define FTR_SIZE 1 // in words
 #define PRED_FIELD_SIZE 1 // in words
+#define HEAP_PADDING 2 // in words
```

Comment:
This is to specify size of epilog. When writing down functionality it is clearer to read.